airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/python_connectors.py

```diff
@@ -22,6 +22,8 @@
 from pipelines.helpers.execution.run_steps import STEP_TREE, StepToRun
 from pipelines.models.steps import STEP_PARAMS, Step, StepResult
 
+PYAIRBYTE_VERSION = "0.10.2"
```

Comment:
Could you please add a comment about why we are pinning the pyairbyte version?

Reply:
added a comment